avm/res/key-vault/vault/tests/e2e/private-endpoint/main.test.bicep

```diff
@@ -99,3 +99,5 @@ module testDeployment '../../../main.bicep' = [for iteration in [ 'init', 'idem'
     enablePurgeProtection: false
   }
 }]
+
+output resourceId string = testDeployment[1].outputs.resourceId
```

Comment:
I'd suggest to remove these tests. But should we maybe add some for a module owned by a maintainer? @ChrisSidebotham / @eriqua?

Reply:
You mean remove all added pe tests? Agree, and sure for adding them to a module owned by a maintainer. Easy for me to say as I don't own one :smile:

Reply:
I anyway guess the real implementation example will come with the first pattern module

Reply:
Got sign-off by @fblix . Beautified & re-running tests